packages/react-components/src/components/CameraButton.tsx

```diff
@@ -170,6 +180,11 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
       menuProps={props.menuProps ?? generateDefaultDeviceMenuPropsTrampoline(props, strings)}
       menuIconProps={props.menuIconProps ?? !enableDeviceSelectionMenuTrampoline(props) ? { hidden: true } : undefined}
       split={props.split ?? enableDeviceSelectionMenuTrampoline(props)}
+      ariaDescription={
```

Comment:
Can you explain more in the PR description what this is doing and why is it needed? Does fluent not provide the aria roles for their buttons already?

Reply:
Wait -- is there a difference between aria-roledescriptiona and aria-description?

Reply:
they dont provide the explicit narration of 'split button' which I think would provide more context to say the drop down is attached to this button, will update for more clarity 😊